spec/spec_helper.rb

```diff
@@ -18,6 +18,8 @@
   config.mock_with :rspec
   config.include Rack::Test::Methods
 
+  config.filter_run_when_matching :focus
```

Comment:
This line for `focus` and to manage this kind of test `it 'raises an error if no requests have been issued', focus: true` is not related to the main issue directly isn't it?
We can create another PR for that.
How do you think?

Reply:
I agree. I actually just left it in the PR to ask you and @perlun for your opinion of having it in our configuration. If it is fine with both of you, I will remove the line and add a new PR.

Reply:
I'm fine with that change. For reference: https://relishapp.com/rspec/rspec-core/docs/filtering/filter-run-when-matching

(We shouldn't have any specs focused in the committed tree, so please remove the `focus: true` in your PR for that also.)